iosHackthonApp/Modules/Features/MealList/Sources/MealList/MealListViewModel.swift

```diff
@@ -40,9 +40,11 @@ public final class MealListViewModel: ObservableObject {
                            completionHandler: { meals, error in
             guard
                 let meals = meals,
-                error == nil else { return }
+                error == nil else {
+                assertionFailure(String(describing: error!.localizedDescription))
```

Comment:
You don't need the `String(describing` bit now since that was just to fix the optional :)

Reply:
Dopey of me 🤦